libretroshare/src/gxs/rsgxsnetservice.cc

```diff
@@ -4231,52 +4223,59 @@ void RsGxsNetService::handleRecvSyncMessage(RsNxsSyncMsgReqItem *item,bool item_
-		    mItem->authorId = m->mAuthorId;
-		    mItem->PeerId(peer);
-		    mItem->transactionNumber = transN;
+                RsNxsSyncMsgItem* mItem = new RsNxsSyncMsgItem(mServType);
+                mItem->flag = RsNxsSyncGrpItem::FLAG_RESPONSE;
+                mItem->grpId = m->mGroupId;
+                mItem->msgId = m->mMsgId;
+                mItem->authorId = m->mAuthorId;
+                mItem->PeerId(peer);
+                mItem->transactionNumber = transN;
 
-		    if(!should_encrypt_to_this_circle_id.isNull())
-		    {
+                if(!should_encrypt_to_this_circle_id.isNull())
+                {
 #ifdef NXS_NET_DEBUG_7
-			    GXSNETDEBUG_PG(item->PeerId(),item->grpId) << "    sending info item for msg id " << mItem->msgId << ". Transaction will be encrypted for group " << should_encrypt_to_this_circle_id << std::endl;
+                    GXSNETDEBUG_PG(item->PeerId(),item->grpId) << "    sending info item for msg id " << mItem->msgId << ". Transaction will be encrypted for group " << should_encrypt_to_this_circle_id << std::endl;
 #endif
-			    RsNxsItem *encrypted_item = NULL ;
-			    uint32_t status = RS_NXS_ITEM_ENCRYPTION_STATUS_UNKNOWN ;
+                    RsNxsItem *encrypted_item = NULL ;
+                    uint32_t status = RS_NXS_ITEM_ENCRYPTION_STATUS_UNKNOWN ;
 
-			    if(encryptSingleNxsItem(mItem, grpMeta->mCircleId,m->mGroupId, encrypted_item,status))
-			    {
-				    itemL.push_back(encrypted_item) ;
-				    delete mItem ;
-			    }
-			    else
-			    {
-				    // Something's not ready (probably the circle content. We could put on a vetting list, but actually the client will re-ask the list asap. 
+                    if(encryptSingleNxsItem(mItem, grpMeta->mCircleId,m->mGroupId, encrypted_item,status))
+                    {
+                        itemL.push_back(encrypted_item) ;
+                        delete mItem ;
+                    }
+                    else
+                    {
+                        // Something's not ready (probably the circle content. We could put on a vetting list, but actually the client will re-ask the list asap.
 
-				    std::cerr << "  (EE) Cannot encrypt msg meta data. MsgId=" << mItem->msgId << ", grpId=" << mItem->grpId << ", circleId=" << should_encrypt_to_this_circle_id << ". Dropping the whole list." << std::endl;
+                        std::cerr << "  (EE) Cannot encrypt msg meta data. MsgId=" << mItem->msgId << ", grpId=" << mItem->grpId << ", circleId=" << should_encrypt_to_this_circle_id << ". Dropping the whole list." << std::endl;
 
-				    for(std::list<RsNxsItem*>::const_iterator it(itemL.begin());it!=itemL.end();++it)
-					    delete *it ;
+                        for(std::list<RsNxsItem*>::const_iterator it(itemL.begin());it!=itemL.end();++it)
+                            delete *it ;
 
-				    itemL.clear() ;
-				    break ;
-			    }
-		    }
-		    else
-		    {
+                        itemL.clear() ;
```

Comment:
The reason is simple: when I review a pull request, I cannot afford to waste time figuring out that modifications are just indentation modifications, which can easily hide small bugs and therefore still need my full attention and time.